integration-tests/src/test/java/org/apache/fineract/integrationtests/common/loans/LoanApplicationTestBuilder.java

```diff
@@ -212,6 +214,12 @@ private String build(final HashMap<String, Object> map, final String loanProduct
         if (enableDownPayment == true) {
             map.put("enableDownPayment", enableDownPayment);
         }
```

Comment:
Please add a test case where down payment was configured on loan product (example: downpayment percentage is 33%), but it got overridden during loan submit (example: downpayment percentage to be 25%) when the loan got submitted, the repayment schedule shows down payment period and 25% was used to calculate the proper amount.

Reply:
Done! ITC case added